### PR TITLE
fix: Fix the bug that the component will be deleted when editing in the Input

### PR DESCRIPTION
**Pull request by xzdry**

### CODE REVIEW

File: packages/editor/src/components/KeyboardEventWrapper.tsx [hunk 1]
```diff
@@ -104,15 +108,17 @@ export const KeyboardEventWrapper: React.FC<Props> = ({
...
-      id="keyboard-event-wrapper"
-      className={style}
-      width="100%"
-      height="100%"
-      onKeyDown={onKeyDown}
-      tabIndex={-1}
-    >
-      {children}
-    </Box>
+    <>
```
Comment (tanbowensg) [inline packages/editor/src/components/KeyboardEventWrapper.tsx:111]: why add this?

File: packages/editor/src/components/KeyboardEventWrapper.tsx [hunk 2]
```diff
@@ -32,6 +32,10 @@ export const KeyboardEventWrapper: React.FC<Props> = ({
     switch (e.key) {
       case 'Delete':
       case 'Backspace':
+        // if `selectionStart` available,think it is textarea or text input
+        if ((e.target as any).selectionStart !== undefined) {
```
Comment (Yuyz0112) [inline packages/editor/src/components/KeyboardEventWrapper.tsx:36]: looks like checking the target's tag name is a more straightforward and easy-to-understand way of doing this?

Comment (xzdry) [reply packages/editor/src/components/KeyboardEventWrapper.tsx:36]: seems better，i'll check it out

Comment (tanbowensg) [reply packages/editor/src/components/KeyboardEventWrapper.tsx:36]: Checking tag name cannot detect whether the input is focused.

**Review by tanbowensg — APPROVED** [review]

File: packages/editor/src/components/KeyboardEventWrapper.tsx (hunk 2 above)
Comment (xzdry) [reply packages/editor/src/components/KeyboardEventWrapper.tsx:36]: > Checking tag name cannot detect whether the input is focused.

It should be possible, the result of my test is that the tagname is `INPUT` only in the focus state

**Pull request merged**